webrtc/videoroom/lib/videoroom/display_manager.ex

```diff
@@ -37,7 +37,7 @@ defmodule VideoRoom.DisplayManager do
   end
 
   @doc """
-  Updates voice activity status of endpoint with id `endpoint_id`.
+  Updates voice activity status of endpoint with id `endpoint_id`. Raises if endpoint doesn't exist.
 
   Returns `{:replace, old_endpoint_id, new_endpoint_id}` if video track of `new_endpoint_id` should be replaced by
```

Comment:
It sound like returning `{:replace, old_endpoint_id, new_endpoint_id}` causes replacing track of `new_endpoint_id` with `old_endpoint_id` track. That's a bit nonintuitive.

Reply:
Yes but it's because we operate on Endpoints both in DisplayEngine and DisplayManager. I am opened for suggestions but I would do it in a separate issue